Stacks_Using_Linked_Lists/Stacks.c

```diff
@@ -63,9 +63,9 @@ void printStack(struct stack *s)
     }
     else
     {
-        printf("\nThe current stack is ");
+        printf("\nThe current stack is\n");
         for (int i = 0; i <= Stack.top; i++)
-        printf("%d\t", s -> data[i]);
+        printf("%d\n", s -> data[i]);
```

Comment:
There was no need to add new line after each element. I just suggested to add a new line after 'The current stack is'.